cartridges/bm_algolia/cartridge/templates/resources/algolia.properties

```diff
@@ -110,3 +110,9 @@ algolia.msg.loading=Loading...
 algolia.msg.confirmclean=Are you sure you want to clean the indexing queue?
 
 algolia.error.service=Service request failed
+algolia.error.missing.permissions=The API key is missing required permissions: {0}
+algolia.error.index.access=The API key does not have access to the required indices
```

Comment:
This is not used and is duplicated by the `algolia.error.index.restrictedprefix` string below...